cl/lib/search_utils.py

```diff
@@ -121,12 +121,12 @@ def get_query_citation(cd: CleanData) -> Optional[List[CaseCitation]]:
         cd["q"], do_post_citation=False, do_defendant=False
     )
 
-    citations = [c for c in citations if isinstance(c, CaseCitation)]
+    citations = [c for c in citations if isinstance(c, FullCaseCitation)]
```

Comment:
It seemed to me that we should actually only be doing this for full cites, not short cites too.